packages/lesswrong/components/posts/PostsPage/ContentType.tsx

```diff
@@ -125,6 +143,10 @@ export const contentTypes = {
       tooltipBody: <div>
         Posts that are relevant to doing good effectively.
       </div>,
+      // ea-forum-lookhere: When "frontpage" or "personal blog" appear in a tags-list,
```

Comment:
This needs to be changed

Reply:
Todo: Probably this is best: https://forum.effectivealtruism.org/about#Finding_content